src/python/pants/option/config.py

```diff
@@ -552,3 +554,70 @@ def get_source_for_option(self, section: str, option: str) -> Optional[str]:
       if cfg.has_option(section, option):
         return cfg.get_source_for_option(section, option)
     return None
+
+
+@dataclass(frozen=True)
+class TomlSerializer:
```

Comment:
Currently, we only use this for `PantsRunIntegrationTest`, but I imagine we might have some use down the road so I put it in `config.py`.